main/inc/ajax/model.ajax.php

```diff
@@ -6,6 +6,10 @@
 
 require_once __DIR__.'/../global.inc.php';
 
+// --- FILTRES DATE ---
+$search_start_date = isset($_REQUEST['start_date']) && !empty($_REQUEST['start_date']) ? $_REQUEST['start_date'] : null;
+$search_end_date = isset($_REQUEST['end_date']) && !empty($_REQUEST['end_date']) ? $_REQUEST['end_date'] : null;
```

Comment:
Those variables are only used in one specific case "get_exercise_pending_results" so they should be initialized only at the beginning of this case (after line 651)